statistics/table.go

```diff
@@ -195,6 +207,8 @@ func (h *Handle) tableStatsFromStorage(tableInfo *model.TableInfo, loadAll bool)
 		// We copy it before writing to avoid race.
 		table = table.copy()
 	}
+	// After loading from storage, table stats becomes not pseudo.
+	table.Pseudo = false
```

Comment:
Why?

Reply:
When we use `planBuilder.getStatsTable()` to get the stats table, we will compute whether the table is pseudo. We don't need to compute it at here, therefore we set the Pseudo of the table false here.